packages/odyssey-react-mui/src/themes/odyssey/shape.ts

```diff
@@ -14,5 +14,5 @@ import type { ThemeOptions } from "@mui/material";
 import * as Tokens from "@okta/odyssey-design-tokens";
 
 export const shape: ThemeOptions["shape"] = {
-  borderRadius: Tokens.BorderRadiusBase,
+  borderRadius: parseInt(Tokens.BorderRadiusBase, 10),
```

Comment:
Not your fault, but I always get antsy when people use `parseInt`. Is there some bad article or StackOverflow for it?

We should be using `Number` instead:
```js
borderRadius: Number(Tokens.BorderRadiusBase),
```

`parseInt` has other issues. It's meant for converting bases rather than type-casting.

Reply:
I'm using `parseInt` to strip the non-numerical content from the Token, which is currently `4px`.

`Number()` will return `NaN` in this case, rather than a unit-stripped number, won't it?

Reply:
I see what you're saying. Yes, `Number()` will be `NaN`.

I remember `parseInt` having issues in the past with automatic stripping like this. I can't remember exactly what caused it, but my suggestion was to use `.replace` instead:
```js
Tokens.BorderRadiusBase.replace(/(\d+)/, '$1')
```

But this doesn't work on numbers. You'd basically have to first check the type.

I understand that using `parseInt` is all-inclusive, but I don't trust it, and I think the solution is clever meaning it's not going to be clear to another dev why it's being used here.

Maybe we need something like:
```js
typeof Tokens.BorderRadiusBase === 'string'
? Tokens.BorderRadiusBase.replace(/(\d+)/, '$1')
: Tokens.BorderRadiusBase
```

While it's more code, it's also more clear.

Reply:
Applied a version of this; slightly different regex to get the desired result